packages/main/src/ColorPalette.ts

```diff
@@ -502,6 +502,12 @@ class ColorPalette extends UI5Element {
 
 	_openMoreColorsDialog() {
 		const dialog = this._getDialog();
+		const colorPicker = this.getColorPicker();
+		const value = this._currentlySelected ? this._currentlySelected.value : undefined;
+
+		if (value) {
+			colorPicker.value = value;
+		}
```

Comment:
isn't this the same as 

colorPicker.value = this._currentlySelected ? this._currentlySelected.value : undefined;

Reply:
Nope, as the colorPicker **value** property cannot accept `undefined` as it is of type **string**  but **_currentlySelected** `value` property is optional.